popx/migration_box_testdata_test.go

```diff
@@ -0,0 +1,36 @@
+package popx_test
+
+import (
+	"embed"
+	"testing"
+
+	"github.com/gobuffalo/pop/v6"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ory/x/logrusx"
+	"github.com/ory/x/popx"
+)
+
+//go:embed stub/migrations/testdata/*
+var testData embed.FS
+
+//go:embed stub/migrations/empty/*
+var empty embed.FS
+
+func TestMigrationBoxWithTestdata(t *testing.T) {
+	c, err := pop.NewConnection(&pop.ConnectionDetails{
+		URL: "sqlite://file::memory:?_fk=true",
+	})
+	require.NoError(t, err)
+	require.NoError(t, c.Open())
+
+	mb, err := popx.NewMigrationBox(
+		empty,
+		popx.NewMigrator(c, logrusx.New("", ""), nil, 0),
+		popx.WithTestdata(testData))
+
+	require.NoError(t, err)
+	assert.Len(t, mb.Migrations["up"], 1)
+	assert.Equal(t, "testdata", mb.Migrations["up"][0].Name)
```

Comment:
You could have some simple `CREATE TABLE` example in the migrations, with the testdata inserting a row. That would be a more realistic test. You can in the end verify whether the row was inserted.